common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -792,32 +792,46 @@ def _get_answer_notification(self, render_notifications):
                         break
 
             # Build the notification message based on the notification type and translate it.
+            data = getattr(self, 'data', '')
+            # Customization for the default system responses while submitting answers in Python-Evaluated Input Block
+            # by `python_evalueted_block` flag. The `data.find('<jsinput') < 0` check provided to exclude Custom
+            # JavaScript Display and Grading block.
+            python_evalueted_block = data.find('<script type="loncapa/python">') >= 0 and data.find('<jsinput') < 0
             ungettext = self.runtime.service(self, "i18n").ungettext
             if answer_notification_type == 'incorrect':
                 if progress is not None:
-                    answer_notification_message = ungettext(
-                        "Incorrect ({progress} point)",
-                        "Incorrect ({progress} points)",
-                        progress.frac()[1]
-                    ).format(progress=str(progress))
+                    if not python_evalueted_block:
+                        answer_notification_message = ungettext(
+                            "Incorrect ({progress} point)",
+                            "Incorrect ({progress} points)",
+                            progress.frac()[1]
+                        ).format(progress=str(progress))
+                    else:
+                        answer_notification_message = _('Answer submission required')
```

Comment:
Also, why don't you use `ungettext`?

Reply:
Messages applied according to the ticket. There are no plural forms for messages.

Reply:
Ok. What about translations? Do we need it?

Reply:
No, we don't need them.